src/main/java/mate/academy/hash/HashUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.hash;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITM = "SHA-256";
+    private static final int SIZE = 32;
+
+    public HashUtil() {
+    }
+
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();
```

Comment:
SecureRandom should be instantiated once.

Think about making it class level field.